.prettierrc

```diff
@@ -0,0 +1,7 @@
+{
+  "semi": true,
+  "tabWidth": 2,
+  "printWidth": 100,
+  "singleQuote": true,
```

Comment:
Am I a terrible person for wanting this to be false? 

Double quotes is the convention that we've been attempting to follow. If we change this to true, perhaps, the amount of churn (change) in this PR will drop a bit.

Reply:
Yes, it gives quite less changes. I will update it. 
![image](https://github.com/user-attachments/assets/d4f72109-4d4e-4921-9c76-8f0c1f04fcad)

Reply:
Request change is done.